src/test/java/seedu/address/logic/commands/FindCommandTest.java

```diff
@@ -55,29 +65,67 @@ public void equals() {
     }
 
     @Test
-    public void execute_zeroKeywords_noPersonFound() {
-        String expectedMessage = String.format(MESSAGE_PERSONS_LISTED_OVERVIEW, 0);
-        NameContainsKeywordsPredicate predicate = preparePredicate(" ");
+    public void execute_oneField_multiplePersonsFound() {
+        String expectedMessage = String.format(MESSAGE_PERSONS_LISTED_OVERVIEW, 2);
+        PersonMatchesKeywordsPredicate
+            predicate = new PersonMatchesKeywordsPredicateBuilder()
+            .withName(KEYWORD_MATCHING_MEIER).build();
```

Comment:
8 spacing indentation for wrap-text lines here (and the remaining lines of this class) also